sentry-core/build.gradle.kts

```diff
@@ -49,4 +56,60 @@ tasks {
         dependsOn(jacocoTestCoverageVerification)
         dependsOn(jacocoTestReport)
     }
+
+    withType<BintrayUploadTask> {
+        bintray {
+            user = System.getenv("BINTRAY_USER")
+            key = System.getenv("BINTRAY_KEY")
+            with(pkg) {
+                repo = "mvn"
+                name = project.name
+                userOrg = "getsentry"
+                setLicenses("MIT")
+                websiteUrl = "https://sentry.io"
+                vcsUrl = "https://github.com/getsentry/sentry-android"
+                setLabels("sentry", "getsentry", "error-tracking", "crash-reporting")
+                with(version) {
+                    name = project.version.toString()
+                    vcsTag = project.version.toString()
+                    desc = project.description.toString()
+                }
+            }
+            println("version: $version")
+            setPublications(project.name)
+        }
+    }
+}
+
+publishing {
+    publications {
+        register(project.name, MavenPublication::class) {
+            if (project.hasProperty("android")) {
+                artifact("$buildDir/outputs/aar/${project.name}-release.aar")
```

Comment:
this is the sentry-core module, we don't need android stuff in here

Reply:
Right, this is all very provisional. It will all move to the main gradle file